### PR TITLE
feat: add firacode

### PR DESCRIPTION
**Pull request by Udacv**
Firacode is a free monospaced font with programming ligatures

Log: add firacode
Issue: deepin-community/sig-deepin-sysdev-team#43

### CODE REVIEW

**Review by Clansty — APPROVED** [review]

**Comment by xzl01** [comment]
不允许有大写包名

File: repos.yml [hunk 1]
```diff
@@ -834,3 +834,8 @@ repos:
     group: deepin-sysdev-team
     info: Rust target for the Windows operating system, used for building Rust applications that run on 64-bit Windows systems using the GNU toolchain.
 
+  - repo: fonts-FiraCode
```
Comment (myml) [inline repos.yml:837]: 包名不能大写

**Review by myml — CHANGES_REQUESTED** [review]

File: repos.yml (hunk 1 above)
Comment (Udacv) [reply repos.yml]: 已修改

**Review by chenchongbiao — APPROVED** [review]

**Pull request merged**